andela/hc-ravens
# Ft running too often 156687758

#### What does this PR do?
- Implements PO feedback to rename a `check` status from `often` to `too often` in the alerts
- Handles bug of check switching from `down` to `too often` instead of `up`

#### Description of Task to be completed
- The client should receive a notification email with **TOO OFTEN** instead of **OFTEN**

#### How should this be manually tested?
- User logs in at [hc-ravens](http://hc-ravens.herokuapp.com/)
- Adds a check and copy the ping URL
- Runs the URL on your browser address bar to activate the Check
- Runs it once more before the timeout period elapses
- Visits [your checks](http://hc-ravens.herokuapp.com/checks/) and you should see an orange bell indicating 'Too often' a job
- User should also receive a notification to their channels

#### Any background context you want to provide?
- None

#### What are the relevant pivotal tracker stories?
#156687758
